docs/coding-guidelines/breaking-change-rules.md

```diff
@@ -236,6 +236,8 @@ successfully bind to that overload, if simply passing an `int` value. However, i
 
     Definite assignment rules allow use of uninitialized variables so long as the variable type is a stateless struct. If the struct is made stateful, code could now end up with uninitialized data. This is both potentially a source breaking and binary breaking change.
 
+* Adding a reference type field, a `ref` field, or a field involving a generic type parameter without the `unmanaged` constraint, to a value type that formerly had none of those field kinds. If the value type already contains at least one such field, adding another is non-breaking.
```

Comment:
It can also be a breaking change to add even a primitive value type, if it's the first field on the type. If the struct previously had no fields, Roslyn might have considered it definitely assigned even if it never was.

Reply:
The existing rule right before this one covers this case.